src/rulesets/reader.ts

```diff
@@ -0,0 +1,36 @@
+import merge = require('lodash/merge');
+import Lint from '../cli/commands/lint';
+import { formatAjv } from '../formatters/ajv';
```

Comment:
Is this added as a formatter because people are expected to also maybe use it like `spectral lint foo.yaml -f ajv` or this just for us? 

If it is just for us, can it not be a formatter? This seems to be slightly confusing the purpose of what formatters are currently doing.

Reply:
Gotcha, it's formatting the `ruleset` errors output. I don't mind renaming it. Any suggestions?